Sources/Run/Models/StripeCharge.swift

```diff
@@ -0,0 +1,80 @@
+//Payment DB model:
+//<fk> customer DB
+//<pk> transaction ID from Stripe
+//<last 4 digits of card> STRING
+//<int> price paid ($1.99 should be 199)
+
+import Foundation
+import FluentProvider
+import Fluent
+import Vapor
+
+final public class StripeCharge: Model {
+    public let storage = Storage()
+    public static let idKey = "charge_id"
+    
+    public var charge_id: String
+    public let payment_info: String
+    public let price: Int
+    public let stripe_customer_id: String
+    
+    public init(charge_id: String, payment_info: String, price: Int, stripe_customer_id: String) {
+        self.charge_id = charge_id
+        self.payment_info = payment_info
+        self.price = price
+        self.stripe_customer_id = stripe_customer_id
+    }
+    
+    public var id: Identifier? {
+        get {
+            return Identifier(.string(self.charge_id), in: nil)
+        }
+        set {
+            self.charge_id = (newValue?.string)!
```

Comment:
p0: Using force unwrap is very unsafe use 
 self.charge_id = newValue?.string ?? ""
or
self.charge_id = newValue?.string ?? "n/a"